terraform/README.md

```
@@ -0,0 +1,36 @@
+## To plan / run specific plays
+
+```shell
+terraform plan
+terraform apply --target=<tf_resourceaddress>
+```
+
+## To destroy a specific play
+
+```shell
+terraform destroy --target=<tf_resourceaddress>
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)